code/modules/projectiles/projectile/magic.dm

```diff
@@ -133,11 +133,18 @@
 			new_mob = new /mob/living/carbon/monkey(M.loc)
 		if("robot")
 			var/robot = pick("cyborg","syndiborg","drone")
+			var/path
 			switch(robot)
 				if("cyborg")
-					new_mob = new /mob/living/silicon/robot(M.loc)
+					if(prob(70))
+						path = /mob/living/silicon/robot
+					else
+						if(config.forbid_secborg)
+							path = /mob/living/silicon/robot/security
+						else
+							path = /mob/living/silicon/robot
```

Comment:
Add a `This is correct` comment on the if other wise somebody will think you forgot a `!` and add it in

Reply:
That's a great point MSO. And probably add to the comment that it is intended for servers where secborg is disabled?

Reply:
I generally just do `this is correct` because I think if the coder isn't able to figure out why, they aren't qualified to be touching that code. however i generally end up fucking with more complex low level systems like the mc or atmos so that's a belief that isn't always true